lighthouse-treemap/app/src/main.js

```diff
@@ -208,6 +211,63 @@ class TreemapViewer {
       };
     }
 
+    /**
+     * @param {LH.Treemap.Node} root
+     * @return {LH.Treemap.ViewMode|undefined}
+     */
+    const createDuplicateModulesViewMode = (root) => {
+      /** @type {Map<string, Array<{node: LH.Treemap.Node, path: string[]}>>} */
+      const moduleNameToNodes = new Map();
+      for (const d1Node of root.children || []) {
+        TreemapUtil.walk(d1Node, (node, path) => {
+          if (node.children) return;
+          if (!node.duplicatedNormalizedModuleName) return;
+
+          const nodes = moduleNameToNodes.get(node.duplicatedNormalizedModuleName) || [];
+          nodes.push({node, path});
+          moduleNameToNodes.set(node.duplicatedNormalizedModuleName, nodes);
+        });
+      }
+
+      let potentialByteSavings = 0;
+
+      /** @type {LH.Treemap.NodePath[]} */
+      const highlightNodePaths = [];
+      for (const nodesWithSameModuleName of moduleNameToNodes.values()) {
+        if (nodesWithSameModuleName.length === 1) continue;
+
+        const bytes = [];
+        for (const {node} of nodesWithSameModuleName) {
+          bytes.push(node.resourceBytes);
+          console.log(node.duplicatedNormalizedModuleName);
+        }
+
+        // Sum all but the largest copy.
+        bytes.sort((a, b) => b - a);
+        let duplicatedBytes = 0;
+        for (let i = 1; i < bytes.length; i++) duplicatedBytes += bytes[i];
+        if (duplicatedBytes < DUPLICATED_MODULES_IGNORE_THRESHOLD) continue;
+
+        for (const {path} of nodesWithSameModuleName) {
+          highlightNodePaths.push([root.name, ...path]);
+        }
+        potentialByteSavings += duplicatedBytes;
+      }
+
+      // TODO: Show button but disable?
```

Comment:
I would prefer this to not showing it personally.

Reply:
yeah I think so too

Reply:
I hope